tests/core/test_processor.py

```diff
@@ -1032,3 +1034,57 @@ async def mocked_run(*args: Any, **kwargs: Any) -> List[Event]:
     ]
     for event, expected in zip(tracker.events, expected_events):
         assert event == expected
+
+
+async def test_restart_session(
+    default_channel: CollectingOutputChannel,
+    default_processor: MessageProcessor,
+    monkeypatch: MonkeyPatch,
+):
+    # The rule policy is trained and used so as to allow the default action ActionRestart to be predicted
+    rule_policy = RulePolicy()
+    rule_policy.train([], default_processor.domain, RegexInterpreter())
+    default_processor.policy_ensemble.policies.insert(0, rule_policy)
```

Comment:
As said in Slack this will manipulate an object which is used by later tests. So it's best to use `monkeypatch` to replace the used `PolicyEnsemble`